<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSceneToolFor2dClass</class>
 <widget class="QWidget" name="CSceneToolFor2dClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSceneToolFor2d</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{font-family:微软雅黑;font-size:14px;color:#ffffff;background:transparent;}
#widget{border-image:url(&quot;:map_manipulator/Resources/images/mainBack.png&quot;);}
#line{border-image:url(&quot;:map_manipulator/Resources/images/line_hor.png&quot;);}
#synchIconWgt{border-image: url(&quot;:map_manipulator/Resources/images/synch.png&quot;);}

QPushButton{padding-bottom: -46px; color:#999999; font-family:微软雅黑; font-size:14px}
QPushButton:hover{color:#ffffff}

#synchPushBtn,#airViewPushBtn,#navigatorPushBtn,#mapScalePushBtn,#graticulePushBtn,#twilightPushBtn,#sceneInfoPushBtn{ border-image: url(:/map_manipulator/Resources/images/switch_off.png)}
#synchPushBtn:checked,#airViewPushBtn:checked,#navigatorPushBtn:checked,#mapScalePushBtn:checked,#graticulePushBtn:checked,#twilightPushBtn:checked,#sceneInfoPushBtn:checked { border-image: url(&quot;:/map_manipulator/Resources/images/switch_on.png&quot;); }
#synchPushBtn:disabled,#airViewPushBtn:disabled,#navigatorPushBtn:disabled,#mapScalePushBtn:disabled,#graticulePushBtn:disabled,#twilightPushBtn:disabled,#sceneInfoPushBtn:disabled { border-image: url(&quot;:/map_manipulator/Resources/images/switch_disabled.png&quot;); }

QLabel:disabled
{
	color:rgba(255, 255, 255, 60%);
}

CSceneToolItem
{
	border-image:url(:map_manipulator/Resources/images/item_bg.png) 5 10 6 6;
	border-width: 5 10 6 6;
	font-family: &quot;Microsoft YaHei&quot;;
}
CSceneToolItem:hover
{
	border-image:url(:map_manipulator/Resources/images/item_bg_hover.png) 5 10 6 6;
}
CSceneToolItem[selected=&quot;true&quot;]
{
	 border-image:url(:map_manipulator/Resources/images/item_bg_hover.png) 5 10 6 6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>238</width>
        <height>407</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="synchIconWgt" native="true">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/map_manipulator/Resources/images/synch.png);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="syncLabel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>view sync</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="synchPushBtn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="airViewIcon" native="true">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/map_manipulator/Resources/images/niaokantu.png);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_airView">
             <property name="text">
              <string>airView</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="airViewPushBtn">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="navigatorIcon" native="true">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/map_manipulator/Resources/images/danghangqi.png);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_navigator">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>navigator</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="navigatorPushBtn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="mapScaleIcon" native="true">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/map_manipulator/Resources/images/bilichi.png);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_mapScale">
             <property name="text">
              <string>mapSacle</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="mapScalePushBtn">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="graticuleIcon" native="true">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/map_manipulator/Resources/images/jingweiwangge.png);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_graticule">
             <property name="text">
              <string>graticule</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="graticulePushBtn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="sceneInfoIcon_4" native="true">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/map_manipulator/Resources/images/changjingxinxi.png);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_twilight">
             <property name="text">
              <string>晨昏线</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_15">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="twilightPushBtn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="sceneInfoIcon" native="true">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/map_manipulator/Resources/images/changjingxinxi.png);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_sceneInfo">
             <property name="text">
              <string>sceneInfo</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="sceneInfoPushBtn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>12</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="toolWgt" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MapManipulator.qrc"/>
 </resources>
 <connections/>
</ui>
